src/app/modules/payments/payments.controller.ts

```diff
@@ -114,3 +115,12 @@ export const handleSendOnboardingEmail: ControllerHandler<
       }
     })
 }
+
+export const handleSendOnboardingEmail = [
+  celebrate({
+    [Segments.BODY]: Joi.object({
+      email: Joi.string(),
```

Comment:
Do we need to add `.required()` and `.email()`?
```suggestion
      email: Joi.string().email().required(),
```